vmdb/app/views/layouts/_edit_to_email.html.haml

```diff
@@ -12,9 +12,11 @@
             = _("None")
           - else
             - if @temp[:email_to].present?
-              - emails = @temp[:email_to].sort { |a, b| a[1] <=> b[1] }.collect { |e| e[0] }
-            - elsif !@edit[:new][:email][:to].present?
-              - emails = @edit[:new][:email][:to].sort { |a, b| a[1] <=> b[1] }
+              - emails = @temp[:email_to]
+                  .sort_by { |_email, display_email| display_name }
+                  .collect { |email, _display_email| email }
+            - elsif @edit[:new][:email][:to].present?
+              - emails = @edit[:new][:email][:to].sort
```

Comment:
Fixed the haml conversion bug @romanblanco .  However, I'd still like a review for the other issue with the sort and if my fix is accurate: https://github.com/ManageIQ/manageiq/pull/1846#discussion-diff-25207905